modules/tls-profiles-understanding.adoc

```diff
@@ -26,7 +26,7 @@ For the Ingress Controller, the minimum TLS version is converted from 1.0 to 1.1
 ====
 
 |`Intermediate`
-|This profile is the recommended configuration for the majority of clients. It is the  default TLS security profile for the Ingress Controller, kubelet, and Kubernetes control plane. The profile is based on the link:https://wiki.mozilla.org/Security/Server_Side_TLS#Intermediate_compatibility_.28recommended.29[Intermediate compatibility] recommended configuration.
+|This profile is the recommended configuration for the majority of clients. It is the  default TLS security profile for the Ingress Controller, kubelet, and control plane. The profile is based on the link:https://wiki.mozilla.org/Security/Server_Side_TLS#Intermediate_compatibility_.28recommended.29[Intermediate compatibility] recommended configuration.
```

Comment:
s/[id="tls-profiles-ingress-understanding_{context}"]/[id="tls-profiles-understanding_{context}"] to match the file name?

S/You can specify one of the following TLS security profiles/You can specify one of the following TLS security profiles for each component (to avoind folks thinking they need to set the same one for each??